tabs/src/components/app/views/RSCList.jsx

```diff
@@ -50,33 +52,44 @@ export function RSCList() {
                         }));
                     setAlert(false);
                     setRSCList(RSCRows);
+                    setIsLoading(false);
                 }
                 else {
                     setAlert(true);
+
                 }
             }
         };
         getRSCList(context);
     }, [context, t]);
 
+    if (isLoading) {
+        return (
+            <Loader
+                size='medium'
+                label={t("RSC.loading")}
+            />
+        );
+    } else {
+        return (
+            <div>
+              {RSCList?.length > 0 && (
+                  <>
+                      <Table className="table" variables={{
+                          cellContentOverflow: 'none',
+                      }}
+                          header={{ items }}
+                          rows={RSCList}
+                          aria-label="RSC Table" />
+                  </>
+              )}
```

Comment:
can we use ternary statements? the project has been using mostly ternary statements